FizzBuzz.py

```diff
@@ -1,27 +1,38 @@
-from types import ListType, IntType
+"""Print all integers from 1 to 100. However, for multiples of three,
+print 'Fizz' instead of the number, for multiples of five, print
+'Buzz', and for numbers which are multiples of both three and five,
+print 'FizzBuzz'.
+"""
+def is_appropriate(number):
+    """Return True if the given number is an integer between 0 and
+    101. Otherwise, return False.
+    """
+    if isinstance(number, int) and 0 < number <= 100:
```

Comment:
code golf:

return isinstance(number, int) and 0 < number <= 100

is equivalent and is branch free (if statement free).  Usually if you find yourself returning True or False, you can simplify to just return the result of the if statement